openshiftcontrolplane/v1/types.go

```diff
@@ -48,6 +51,20 @@ type OpenShiftAPIServerConfig struct {
 	APIServerArguments map[string][]string `json:"apiServerArguments"`
 }
 
+type FrontProxyConfig struct {
+	// ca is a path to the CA bundle to use to verify the front proxy's client cert
+	CA string `json:"ca"`
+	// allowedNames is an optional list of common names to require a match from.
+	AllowedNames []string `json:"allowedNames"`
```

Comment:
common names of what? The presented client cert?

Reply:
> common names of what? The presented client cert?

yeah, will update

Reply:
Lgtm then.